packages/drivers/routerlicious-urlResolver/package.json

```diff
@@ -88,9 +70,11 @@
 		"prettier": "prettier --check . --cache --ignore-path ../../../.prettierignore",
 		"prettier:fix": "prettier --write . --cache --ignore-path ../../../.prettierignore",
 		"test": "npm run test:mocha",
-		"test:mocha": "mocha --ignore \"dist/test/types/*\" --recursive dist/test -r node_modules/@fluid-internal/mocha-test-setup",
+		"test:mocha": "npm run test:mocha:esm && echo skipping cjs to avoid overhead - npm run test:mocha:cjs",
```

Comment:
This is using a package that is "special" compared to prior CJS version. Hopefully it handles both, but we should test both.

Reply:
why is this special?